src/sharechain/p2chain_level.rs

```diff
@@ -142,6 +146,8 @@ impl<T: BlockCache> P2ChainLevel<T> {
             target_difficulty: block.target_difficulty(),
             total_pow: block.total_pow(),
             verified: block.verified,
+            wallet_address_base58: block.miner_wallet_address.to_base58(),
```

Comment:
Is storing it as a tariaddress not more efficient?

Reply:
no, this is only used in string form, as the `get_coinbases` grpc method takes a string